ls1intum/Artemis
# `Assessment`: Fix the refresh button on the exercise scores page

### Checklist
#### General
#### Client

### Motivation and Context
The refresh button on the exercise scores page does not work, the showed results stays the same.

### Description
The button only changed the `result` field, not `filteredResults`. I move the code setting this field into a reusable method.

### Steps for Testing
Prerequisites:
- 1 Instructor
- 1 Student
- 1 Exercise of any type (best with programming exercises due to automatic results)

1. Instructor: Open the Scores page of the exercise
2. Student: Submit a new version of the exercise (let the test finish for PE)
3. Instructor: Press the refresh button
4. Instructor: Check that the new test result is now displayed. 

### Review Progress
#### Code Review
#### Manual Tests
